src/OSSupport/IsThread.cpp

```diff
@@ -110,14 +109,9 @@ bool cIsThread::Start(void)
 
 void cIsThread::Stop(void)
 {
-	if (!m_Thread.joinable())
-	{
-		// The thread hasn't been started or has already been joined
-		return;
-	}
```

Comment:
I've also removed this because it's redundant, `Wait` already checks `m_Thread.joinable()`.